web/src/components/Header/index.css

```diff
@@ -0,0 +1,9 @@
+.ant-popover-title {
```

Comment:
Let's add this as part of the main styled component here `web/src/components/Header/Header.styled.tsx`

```
export const Header = styled(Layout.Header)`
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 24px;
  background: #fff;
  border-bottom: 1px solid #e2e4e6;
  
  .ant-popover-title {
    padding: 16px;
}
.ant-popover-arrow {
    display: none;
}
.ant-popover-inner-content {
    padding: 0;
}
`;
```

Reply:
this does not work.
I have tried this